roles/people-ops.md

```diff
@@ -39,7 +51,12 @@ I've also created your onboarding checklist so you can continue with the onboard
 %0D%0A
 Regards%0D%0A%0D%0A
 "> account credentials </a> email.
-* Once email encryption is setup, send the <a href="mailto:@niteo.co?
+
+
+#### Trialist checklists
+
+1. Create a new [trialist welcome issue] and assign it to the new team member.
+    * Once email encryption is set up, send the <a href="mailto:@niteo.co?
```

Comment:
I would throw encrypted email out and add signal. It is a lot easier to setup and we already decided that we will start using it.

Reply:
That will be done as part of https://github.com/niteoweb/operations/issues/448.